feedback-rep2004.md

```diff
@@ -0,0 +1,44 @@
+- Lack of granularity in ‘quality level’
+While having a few 'quality levels' is great to get a very quick overview of a package quality, it may also be misleading, masking the granularity of reality. A package that excels in all metrics but performs miserably in a single one (for any reason) may be ranked in one of the lower grades thus not encouraging its use. In reality the package may only need a little external help to get a full score.
+An example of this is a package not available on one of the Tier1 platforms, thus not qualifying to be quality level < 4. But being production ready and complying with every other criterium to be level 1.
+We think adding a full 'report card' could be useful to assess how a package is failing to meet criteria for the higher quality level. For this 'report card' to be useful, it should be quickly and easily readable. We could consider a 'spider chart' (or [radar chart as per wikipedia](https://en.wikipedia.org/wiki/Radar_chart)), or a table like [here](https://github.com/ros-infrastructure/rep/blob/c8bd456f531acc4863d0bb8888c28b10f9271492/rep-2004.rst#quality-level-comparison-chart).
+
+### Security specific context and feedback
+
+#### Quality Levels and the Vulnerability Disclosure Policy
+The VDP tells the public we’d like a flat 90-day grace period between when we receive a vulnerability before it’s made public.  In the interest of keeping the VDP simple, I recommend keeping code quality out of this public-facing document.  We can--and should--strive to do better for high quality code, but I don’t believe that needs to be a part of this document.
+
+#### Quality Levels and Vulnerability Remediation
+Internally, however, we should set a higher standard for higher quality code.
+Level 1 and 2 code will be widely deployed in the field. Fixes should be made available this code quickly to protect production robots.
+Level 3 and some level 4 code may not be deployed in robots, but will be used daily by firms that build robots.  High risk vulnerabilities should be handled quickly to protect development environments.
+CVSS is the defacto standard for risk assessing vulnerabilities.  Although flawed, it’s better than trying to create a new standard.
+
+#### Remediation Timeline
+Here’s an initial proposal on responsiveness based on the principles above.  This would be a commitment by the ROS 2 developer community.
+Triage within 3 days for all reports, triage assigns a CVSS score and a maintainer to provide a fix
+Maintainer is expected to provide a fix according to this timeline; the maintainer may also change the CVSS score as they dig into the issue:
+
+
+|                   | High Risk<br> CVSS 9+ | Medium Risk<br> CVSS 6-9            | Low Risk<br>CVSS <6                       |
```

Comment:
How did we come up with this? I'd suggest we apply more security standardized categorization. See Appendix A in https://arxiv.org/pdf/1807.10357.pdf for the following:

<img width="800" alt="imagen" src="https://user-images.githubusercontent.com/1375246/76689537-20ef2780-6637-11ea-8111-251c2141c3c6.png">

How about following those 5 categories? Let me know what you think, happy to send adjustments accordingly.

Reply:
That's a good point, using the official scale makes sense.
I'll let @SidFaber comment to avoid assumptions on the rationale for using the scale proposed here